### ISSUE TITLE
SyntaxError: Unexpected identifier

### ISSUE THREAD

**Issue by macmichael01** [terkelg/prompts#124]
I ran this little snippet from the terminal and got the following error: 

```
const prompts = require('prompts');
(async function () {
let response = await prompts({
  type: `text`,
  name: `desc`,
  message: `description`,
  initial: ``
});
console.log(response);
})();
```

? Project description › This is a test
This is a test
     ^^
SyntaxError: Unexpected identifier

prompts@2.0.1
Bash 4.4.23
Node v11.2.0

**Comment by terkelg** [comment]
Thanks for the PR. Does this happen when you run in from a script or is it only in the REPL?

**Comment by DrunkenPoney** [comment]
That's weird... I tried exactly the same and... well... a picture is worth a thousand words:

![weird](https://i.imgur.com/wlO9akr.png)

It throws an error which is not a `SyntaxError` as mentioned by @macmichael01 but a `ReferenceError`. But anyway it seems like prompts doesn't care about it and return the right answer like if nothing went wrong.

Though, I think I understand what's going on and why it behaves like this:

It seems like both node and the prompts process use the same input source and node REPL doesn't wait until prompts as finished before continuing.

In other words, the node REPL and the prompts process use the same input source and since prompts is async the node REPL wont wait until prompts has finished before continuing.

So I don't think this issue can be solve until there's a way to block user input only for the node REPL but not for prompts process or until there is a way to use prompts synchronously so the node REPL can wait for prompts to finish.
